.github/workflows/build-on-main.yaml

```diff
@@ -31,16 +39,25 @@ jobs:
         if: always()
         with:
           check_name: test-results
-          comment_title: Test Results
+          comment_title: Test Results (${{ steps.install-dotnet.outputs.DPKG_ARCH }})
           files: |
             TestResults/*.trx
 
   snapcraft-build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - [self-hosted, large, jammy, X64]
+          - [self-hosted, large, jammy, ARM64]
```

Comment:
question: How do we extend this eventually to `ppc64el` and `s390x`? There are no self-hosted GitHub runners for `ppc64el` and `s390x` as far as I am aware.

Reply:
Yes, there are no `s390x` and `ppc64el` self-hosted GitHub runners. I believe that we would need to move the builds to Launchpad/Snapcraft CI, but even then we would also need to move the content snap builds as well. It should be easy enough for this repo, but I'm not sure about the content snaps repo, since it contains multiple snap definitions in it.